site/docs/v1/tech/account-management/sms-two-factor.adoc

```diff
@@ -46,10 +46,10 @@ Next,
 
 1. Enter your phone number.
 2. Click on  `Send a one-time-code`.
-3. Enter the `Verification Code` 
+3. Enter the `Verification Code`
 4. Click `Enable`.
 
-=== Recovery Codes (User Facing)
+=== Recovery Codes
```

Comment:
You could probably extract this section out to a `_include` file because it is the same across all the MFA methods. Not a requirement, just a suggestion.